worker/src/main/java/gov/cms/ab2d/worker/processor/coverage/CoverageDriverImpl.java

```diff
@@ -496,4 +498,51 @@ private void checkCoveragePeriodValidity(Job job, CoveragePeriod period) {
     public CoveragePagingResult pageCoverage(CoveragePagingRequest request) {
         return coverageService.pageCoverage(request);
     }
+
+    @Override
+    public void verifyCoverage() {
+
+        List<String> issues = new ArrayList<>();
+
+        List<Contract> enabledContracts = pdpClientService.getAllEnabledContracts().stream()
+                .filter(contract -> !contract.getContractNumber().startsWith("Z"))
```

Comment:
use
`!contract.isTestContract()`  instead.  Keying off the contract name is a hack I previously got rid of.